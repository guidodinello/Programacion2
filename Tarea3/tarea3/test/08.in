# implementar las operaciones de iterador
estaDefinidaActual
agregarAIterador 5
agregarAIterador 2
agregarAIterador 1
agregarAIterador 8
reiniciarIterador
estaDefinidaActual
actualEnIterador
avanzarIterador
avanzarIterador
avanzarIterador
estaDefinidaActual
actualEnIterador
avanzarIterador
estaDefinidaActual
reiniciarIterador
estaDefinidaActual
actualEnIterador
Fin