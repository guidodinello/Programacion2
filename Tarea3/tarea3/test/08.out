1>#  implementar las operaciones de iterador.
2>Actual de it NO está definida.
3>Agregando.
4>Agregando.
5>Agregando.
6>Agregando.
7>Reiniciado.
8>Actual de it está definida.
9>5
10>Avanzando.
11>Avanzando.
12>Avanzando.
13>Actual de it está definida.
14>8
15>Avanzando.
16>Actual de it NO está definida.
17>Reiniciado.
18>Actual de it está definida.
19>5
20>Fin.
